pkg/plugins/platform/local.go

```diff
@@ -588,21 +575,28 @@ func (l *LocalApp) Start() error {
 	return nil
 }
 
-// Exec executes a given command in the container of given type.
-func (l *LocalApp) Exec(service string, tty bool, cmd ...string) error {
+// Exec executes a given command in the container of given type without allocating a pty
+// Returns ComposeCmd results of stdout, stderr, err
+func (l *LocalApp) Exec(service string, cmd ...string) (string, string, error) {
 	l.DockerEnv()
 
-	var exec []string
-	if tty {
-		exec = []string{"exec", "-T", service}
-	} else {
-		exec = []string{"exec", service}
-	}
+	exec := []string{"exec", "-T", service}
 	exec = append(exec, cmd...)
 
 	return dockerutil.ComposeCmd(l.ComposeFiles(), exec...)
 }
 
+// ExecWithTty executes a given command in the container of given type. It allocates a pty for interactive work.
+// Returns ComposeCmd results of err, stdout, stderr
```

Comment:
this comment line seems inaccurate for this function

Reply:
Fixed in 7eb90e2f595dce3c6b1bc27179b1f3fa53428cf3